script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -819,56 +831,56 @@ I've returned to my normal life.
 [sha:a245e3ece1a585d1e4b8039a91cd294c363f0878]{
 -- Page 491, Offset 15978.
 -- 吸血鬼とは関わらない。それは先輩とも約束した事だ。
-I'm not going to involve myself with vampires. I even promised Senpai that.
+I'm not going to get involved with vampires again.
+ I even promised Senpai I wouldn't.
 }
 [sha:9b69ab9f416cf054b3242b7b869779ab57112490]{
 -- Page 492, Offset 15979. Mods: @n.
 -- 「そっか。
-"Oooh well.
+"No dice, huh.
 }
 [sha:85a60d0db8fb177e4a9875147e9c98c7794fed2b]{
 -- Page 492, Offset 15980. Glued.
 -- じゃあとっておきを言ってあげる。
-// could use some fine-tuning, but I think this way of phrasing it gets the best nuance.
-I've been holding back on saying this, but here goes.
+ Well, I wasn't going to bring it up, but...
 }
 [sha:7339f7c8d3b2afa6f742111ede081f9834992426]{
 -- Page 492, Offset 15981.
 -- “お願いだからヴローヴを倒してくれ”
 // reference to Offset 12250
-'Please defeat Vlov'
+'Please defeat Vlov for me.'
 }
 [sha:c732bc0a2b2fbb4f8c3e02f3348f7c9f29fe95c6]{
 -- Page 492, Offset 15982.
 -- これって誰の言葉だったかしら？」
-Oh, I wonder who was it that said these words?"
+Just who was it that said those words, I wonder?"
 }
 [sha:e93a6ff10df7727248cc2a7499a1e5f1181d70be]{
 -- Page 493, Offset 15983.
 -- 「………………」
 "........."
 }
-[offset:15984]{
+[sha:1dd0e0d519069a50030e989c736adbc5fafecc48]{
 -- Page 493, Offset 15984.
 -- それを言われると弱い。
-I can't fight back when she puts it like that.
+She's got me there.
 }
 [sha:2d4e6710425f9245fc5e9c0060981287862f8a2e]{
 -- Page 493, Offset 15985.
 -- アルクェイドはシエル先輩を見てヘソを曲げたものの、しっかり俺たちを助けてくれた。
-Arcueid might've gotten mightily upset after she spotted Ciel-senpai,
- but she still helped us out.
+Arcueid might've flipped out when she spotted Ciel-senpai,
+ but at the end of the day she still helped us.
 }
 [sha:5055e925368ac447d9b143f4ad33971b870db683]{
 -- Page 494, Offset 15986.
 -- あの時アルクェイドが寒波を防いでくれなかったら、俺だけじゃなく街全体が氷漬けになっていた。
-If back then, Arcueid hadn't redirected the cold wave,
- then not only me, but the whole city would've frozen over.
+If she hadn't been there to protect us against the cold wave,
+ I'm not the only one who'd have ended up as a popsicle―――the whole city would have been frozen solid.
```

Comment:
Is the popsicle comparison really necessary?

Reply:
I'm with Petrikow, feels like a weird liberty